edocumentation-transfer-obj/src/main/java/org/dcsa/edocumentation/transferobjects/BookingTO.java

```diff
@@ -151,4 +165,13 @@ public record BookingTO(
 ) {
   @Builder(toBuilder = true)
   public BookingTO { }
+
+  public BookingTO forUpdates() {
+    return toBuilder()
+      .carrierBookingRequestReference(null)
+      .documentStatus(null)
+      .bookingRequestCreatedDateTime(null)
+      .bookingRequestUpdatedDateTime(null)
```

Comment:
If we used the `READ_ONLY` attribute, we would not need to set them to null explicitly (also, if anything, we should probably throw an error if they are not-null).

Reply:
Ok, now throwing an error instead of ignoring them